pyomo/contrib/mindtpy/initialization.py

```diff
@@ -136,6 +141,7 @@ def init_rNLP(solve_data, config):
         # Add OA cut
         # This covers the case when the Lower bound does not exist.
         # TODO: should we use the bound of the rNLP here?
+        # TODO: ipopt will not provide the LB or UB, replce lower_bound and upper_bound with value(obj)
```

Comment:
Here, we use the bound information of the relaxed NLP problem. In Pyomo, `ipopt` will provide the bound information in the results. I am not sure if this applies to all local solvers. Should we change here?
Ref: https://github.com/Pyomo/pyomo/issues/1671

Reply:
There is a typo in this line (replace). Moreover in the case that we are not able to solve the rNLP then the least important problem is that we do not have the bounds. Use as in the Issue suggest the `pyo.value(model.obj)` in the objective and the lower_bound